assets/scss/_custom.scss

```diff
@@ -737,10 +739,17 @@ body.td-documentation {
   padding-top: 40px;
 }
 
+// Always show announcements when javascript is not available
+
+html.no-js body div#announcement {
+  display: block;
```

Comment:
:thought_balloon: Actually, do we mind if the banners are only shown to people who have JavaScript enabled?

That's still nearly everyone&hellip;